tsconfig.json

```diff
@@ -1,15 +1,18 @@
 {
   "compilerOptions": {
-    "declaration": true,
+    "declaration": false,
     "importHelpers": true,
     "module": "commonjs",
     "outDir": "lib",
     "rootDir": "src",
     "strict": true,
     "target": "es2017",
-    "allowJs": true
+    "skipLibCheck": true
   },
   "include": [
-    "src/**/*", "src/typings/@contentstack/cli-command.ts"
+    "src/**/*"
+  ],
+  "exclude": [
```

Comment:
Also need to understand why are we adding `.d.ts` files in exclude section. Basically, we will not have any `.d.ts` file in the src directory